libs/gltfio/src/AssetLoader.cpp

```diff
@@ -53,6 +53,8 @@ using namespace filament;
 using namespace filament::math;
 using namespace utils;
 
+static const auto FREE_CALLBACK = [](void* mem, size_t, void*) { free(mem); };
```

Comment:
shouldn't that be in the gltfio namespace?

Reply:
Fixed.